Fatras/include/ActsFatras/EventData/DigitizedHit.hpp

```diff
@@ -0,0 +1,82 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Surfaces/Surface.hpp"
+#include "ActsFatras/EventData/Hit.hpp"
+
+namespace ActsFatras {
+
+/// A digitizeed hit on a surface.
+///
+/// This is the digitized hit, output of any digitization module. It serves
+/// as source_link in the Surface based Measurement classes and thus has to
+/// fullfill the SourceLinkConcept.
+///
+class DigitizedHit {
+ public:
+  /// @brief Nested IContent base class
+  ///
+  /// This will be overwritten by specialisations of the
+  /// varius digitisers that will know their own content
+  class IContent {
+   public:
+    virtual ~IContent() = default;
+  };
+
+  DigitizedHit() = default;
+
+  /// Constructor with arguments
+  ///
+  /// @param hits The simulated hits at input
+  /// @param surface The surface for the measuremetn representation
```

Comment:
The `measuremetn` should be `measurement`.

Reply:
Fixed.